stdlib/tests/testdata/meta_query_keys.flux

```diff
@@ -37,32 +37,22 @@ outData = "
 ,result,table,_field,_measurement,cpu,host,_value
 ,,0,usage_guest,cpu,cpu-total,host.local,_field
 ,,0,usage_guest,cpu,cpu-total,host.local,_measurement
-,,0,usage_guest,cpu,cpu-total,host.local,_start
-,,0,usage_guest,cpu,cpu-total,host.local,_stop
 ,,0,usage_guest,cpu,cpu-total,host.local,cpu
 ,,0,usage_guest,cpu,cpu-total,host.local,host
 ,,1,usage_guest_nice,cpu,cpu-total,host.local,_field
 ,,1,usage_guest_nice,cpu,cpu-total,host.local,_measurement
-,,1,usage_guest_nice,cpu,cpu-total,host.local,_start
-,,1,usage_guest_nice,cpu,cpu-total,host.local,_stop
 ,,1,usage_guest_nice,cpu,cpu-total,host.local,cpu
 ,,1,usage_guest_nice,cpu,cpu-total,host.local,host
 ,,2,usage_idle,cpu,cpu-total,host.local,_field
 ,,2,usage_idle,cpu,cpu-total,host.local,_measurement
-,,2,usage_idle,cpu,cpu-total,host.local,_start
-,,2,usage_idle,cpu,cpu-total,host.local,_stop
 ,,2,usage_idle,cpu,cpu-total,host.local,cpu
 ,,2,usage_idle,cpu,cpu-total,host.local,host
 ,,3,usage_iowait,cpu,cpu-total,host.local,_field
 ,,3,usage_iowait,cpu,cpu-total,host.local,_measurement
-,,3,usage_iowait,cpu,cpu-total,host.local,_start
-,,3,usage_iowait,cpu,cpu-total,host.local,_stop
 ,,3,usage_iowait,cpu,cpu-total,host.local,cpu
 ,,3,usage_iowait,cpu,cpu-total,host.local,host
 ,,4,usage_irq,cpu,cpu-total,host.local,_field
 ,,4,usage_irq,cpu,cpu-total,host.local,_measurement
-,,4,usage_irq,cpu,cpu-total,host.local,_start
-,,4,usage_irq,cpu,cpu-total,host.local,_stop

```

Comment:
Instead of changing the output of the test here, can you instead add _start and _stop to the group key in the input data as that is closer to the way normal meta queries will work.